### DISCUSSION TITLE
After upgrading to cypress 13, all test fail with blank page after the first IT

### DISCUSSION THREAD

**Discussion by albanx** [cypress-io/cypress#27714] [Questions and Help]
I was using cypress 6, and I decided to upgrade to the latest version. After this all test start failing by showing a blank page.

I tried out the to set the `testIsolation` to false, but that start mixes different tests, without clearing the browser cache.

I do not understand why under the same `it` the test should be in isolation.
A test file should run in isolation but not a single `it` inside the same tests.

Is there a way to preserve the old behavouir?

**Comment by easaborit** [comment]
I happen the same behavior
